New app notification not always shows on the tests pr build

New version notification not always shows on the tests pr build. The response is correct
{
    "message": "Sent a new version available message to 32 HTTP sessions and 64 Vaadin UIs"
}
https://jenkins.dev.devpathmind.com/job/Test_PRs/1874/Tests/87c829d8e2d07b49e14934bf4c83104cb5893633683b1223db4c74ee8ff8d5e9.html
https://jenkins.dev.devpathmind.com/job/Test_PRs/1874/console

@notifications
  Scenario: Check that notification exist after page refresh [90m# src/test/resources/features/notifications.feature:41[0m
    [32mGiven [0m[32mLogin to the pathmind[0m                              [90m# LoginPageStepDefinitions.loginToThePathmind()[0m
http://pathmind-1874/api/newVersionAvailable
HTTP/1.1 200 
X-Content-Type-Options: nosniff
X-XSS-Protection: 1; mode=block
Cache-Control: no-cache, no-store, max-age=0, must-revalidate
Pragma: no-cache
Expires: 0
X-Frame-Options: DENY
Content-Type: application/json
Transfer-Encoding: chunked
Date: Fri, 24 Jul 2020 13:48:12 GMT
Keep-Alive: timeout=60
Connection: keep-alive

{
    "message": "Sent a new version available message to 11 HTTP sessions and 22 Vaadin UIs"
}
    [32mWhen [0m[32mTrigger API new version notification[0m                [90m# ApiStepDefinitions.triggerApiNewVersionNotification()[0m
    [31mWhen [0m[31mCheck that new version notification is shown[0m        [90m# NotificationsStepDefinitions.checkThatNewVersionNotificationIsShown()[0m
      [31mjava.lang.AssertionError: 
Expected: is <1>
     but: was <0>

--- a/pathmind-bdd-tests/src/test/resources/features/notifications.feature
+++ b/pathmind-bdd-tests/src/test/resources/features/notifications.feature
@@ -1,53 +1,53 @@
 @notifications
 Feature: Notifications
 
-#  Scenario: Check new version notification is shown
-#    Given Login to the pathmind
-#    When Trigger API new version notification
-#    Then Check that new version notification is shown
-#
-#  Scenario: Check new version notification text
-#    Given Login to the pathmind
-#    When Trigger API new version notification
-#    Then Check that new version notification text is Pathmind has been updated. Please log in again to get the latest improvements.
-#
-#  Scenario: Check that notification exist after surfing webapp
-#    Given Login to the pathmind
-#    When Trigger API new version notification
-#    When Check that new version notification is shown
-#    When Create new CoffeeShop project with single reward function
-#    When Check that new version notification is shown
-#    Then Click project start run button
-#    When Check that new version notification is shown
-#    When Open projects page
-#    Then Check that new version notification is shown
-#    When Open project AutotestProject on projects page
-#    Then Check that new version notification is shown
-#    Then Click the model name 1
-#    Then Check that new version notification is shown
-#    Then Click the experiment name 1
-#    Then Check that new version notification is shown
-#    When Click in 'Stop Training' button
-#    Then Check that the 'Stop Training' confirmation dialog is shown
-#    When In confirmation dialog click in 'Stop Training' button
-#    Then Check that new version notification is shown
-#
-#  Scenario: Check that notification exist after page refresh
-#    Given Login to the pathmind
-#    When Trigger API new version notification
-#    When Check that new version notification is shown
-#    When Refresh page
-#    Then Check that new version notification is shown
-#
-#  Scenario: Check that there are only one notification
-#    Given Login to the pathmind
-#    When Trigger API new version notification
-#    When Trigger API new version notification
-#    When Trigger API new version notification
-#    Then Check that new version notification is shown
-#
-#  Scenario: Check new version notification sign out btn
-#    Given Login to the pathmind
-#    When Trigger API new version notification
-#    When Click in notification Sign out button
-#    Then Check that login page opened
+  Scenario: Check new version notification is shown
+    Given Login to the pathmind
+    When Trigger API new version notification
+    Then Check that new version notification is shown
+
+  Scenario: Check new version notification text
+    Given Login to the pathmind
+    When Trigger API new version notification
+    Then Check that new version notification text is Pathmind has been updated. Please log in again to get the latest improvements.
+
+  Scenario: Check that notification exist after surfing webapp
+    Given Login to the pathmind
+    When Trigger API new version notification
+    When Check that new version notification is shown
+    When Create new CoffeeShop project with single reward function
+    When Check that new version notification is shown
+    Then Click project start run button
+    When Check that new version notification is shown
+    When Open projects page
+    Then Check that new version notification is shown
+    When Open project AutotestProject on projects page
+    Then Check that new version notification is shown
+    Then Click the model name 1
+    Then Check that new version notification is shown
+    Then Click the experiment name 1
+    Then Check that new version notification is shown
+    When Click in 'Stop Training' button
+    Then Check that the 'Stop Training' confirmation dialog is shown
+    When In confirmation dialog click in 'Stop Training' button
+    Then Check that new version notification is shown
+
+  Scenario: Check that notification exist after page refresh
+    Given Login to the pathmind
+    When Trigger API new version notification
+    When Check that new version notification is shown
+    When Refresh page
+    Then Check that new version notification is shown
+
+  Scenario: Check that there are only one notification
+    Given Login to the pathmind
+    When Trigger API new version notification
+    When Trigger API new version notification
+    When Trigger API new version notification
+    Then Check that new version notification is shown
+
+  Scenario: Check new version notification sign out btn
+    Given Login to the pathmind
+    When Trigger API new version notification
+    When Click in notification Sign out button
+    Then Check that login page opened

